app/controllers/flights_controller.rb

```diff
@@ -2,4 +2,41 @@ class FlightsController < ApplicationController
   def index
     @flights = Flight.all
   end
+
+  def filter
+    return render_flights_list unless flights.empty?
+
+    render_empty_list_message
+  end
```

Comment:
is it possible to move the logic of this action to index and only use that action? Creating a separate route for filtered queries is not very RESTful

Reply:
done!